core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -25,34 +24,50 @@
 
```

Comment:
It isn't the current screen. Game creates has it's own screen and renders to current frame rate rather than the current screen rate. Here are the delta time comparisons with having game render's and without.  
**With**
<img width="571" alt="screen shot 2016-11-14 at 12 43 29 pm" src="https://cloud.githubusercontent.com/assets/16127179/20277919/4dbeebc6-aa68-11e6-94e1-89e3c92158a0.png">
**Without:**
<img width="530" alt="screen shot 2016-11-14 at 12 44 32 pm" src="https://cloud.githubusercontent.com/assets/16127179/20277937/5cdff942-aa68-11e6-9984-229ece1c98ec.png">
There is a clear difference.